apps/dashboard/src/contract-ui/tabs/split/page.tsx

```diff
@@ -44,9 +44,15 @@ export const ContractSplitPage: React.FC<SplitPageProps> = ({
   const configuredChainsRecord = useSupportedChainsRecord();
   const chainId = useDashboardEVMChainId();
   const chain = chainId ? configuredChainsRecord[chainId] : undefined;
-
   const splitQuery = useSplitData(contractQuery.contract);
-  const nativeBalanceQuery = useBalanceForAddress(contractAddress);
+
+  const convertedChain =
+    chainId && chain ? { id: chainId, rpc: chain?.rpc[0] } : undefined;
```

Comment:
should be able to use the `defineDashboardChain` helper function we use in other places in dashboard for this